apps/judicial-system/backend/src/app/modules/subpoena/limitedAccessSubpoena.controller.ts

```diff
@@ -82,4 +83,38 @@ export class LimitedAccessSubpoenaController {
 
     res.end(pdf)
   }
+
+  @RolesRules(defenderGeneratedPdfRule)
+  @Get('serviceCertificate')
+  @Header('Content-Type', 'application/pdf')
+  @ApiOkResponse({
+    content: { 'application/pdf': {} },
+    description:
+      'Gets the service certificate for a given defendant as a pdf document',
+  })
+  async getServiceCertificatePdf(
+    @Param('caseId') caseId: string,
+    @Param('defendantId') defendantId: string,
+    @Param('subpoenaId') subpoenaId: string,
+    @CurrentCase() theCase: Case,
+    @CurrentDefendant() defendant: Defendant,
+    @CurrentSubpoena() subpoena: Subpoena,
+    @Res() res: Response,
+  ): Promise<void> {
+    this.logger.debug(
+      `Getting service certificate for defendant ${defendantId} in subpoena ${subpoenaId} of case ${caseId} as a pdf document`,
+    )
+
+    if (!subpoena) {
```

Comment:
Note that there is an alternative subpoena guard that takes care of this. Please consider moving the `SubpoenaExistsOptionalGuard` to the `getSubpoenaPdf` endpoint and use `SubpoenaExistsGuard` for this endpoint. That will allow you to remove this check.